### ISSUE TITLE
Generated Merge Request link description is invalid

### ISSUE THREAD

**Issue by c33s** [phly/keep-a-changelog#36]
```
- [#130](https://gitlab.com/username/project/merge_requests/130) My New Entry
```
should be:
```
- [!130](https://gitlab.com/username/project/merge_requests/130) My New Entry
```
in gitlab issues and merge requests differ. merge requests are prefixed with `!` like `!130`, issues are prefixed with `#` like on github.

for gitlab it is also relevant to link to merge requests **or** issues or maybe both.

**Comment by weierophinney** [comment]
I'm working on a fix for the markup now, and will have a pull request shortly.

In terms of differentiating between issues and merge requests, the reason the various `entry:*` commands only accept the `--pr` argument is because typically an _issue_ will not make changes to code, while a _patch_ will. As such, I will only address the markup syntax at this time.

**Comment by weierophinney** [comment]
This was fixed in 1.6.0.
